client/components/Content/index.js

```diff
@@ -0,0 +1,2 @@
+import Content from "./Content";
+export default Content;
```

Comment:
IMO, having separate folders is easier to understand/view the components which are under the folder. Having a flat structure is confusing .

Once `export * from 'component` is ready we wont have this issue 💃